### PR TITLE
Populate board

### PR DESCRIPTION
**Pull request by helloramon**
Starting pieces created to populate on new game.

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -5,3 +5,46 @@ class Game < ActiveRecord::Base
   delegate :kings, :queens, :bishops, :knights, :rooks, :pawns, to: :pieces
 
 end
+class Game < ActiveRecord::Base
```
Comment (kenmazaika) [inline app/models/game.rb]: Should not redefine the game class.  This should be tucked into the same game class a the top.

File: app/models/game.rb [hunk 2]
```diff
@@ -5,3 +5,46 @@ class Game < ActiveRecord::Base
...
+
+has_many :users
+has_many :pieces
+delegate :kings, :queens, :bishops, :knights, :rooks, :pawns, to: :pieces
```
Comment (kenmazaika) [inline app/models/game.rb]: You might want to update the indentation.

File: app/models/game.rb [hunk 3]
```diff
@@ -5,3 +5,46 @@ class Game < ActiveRecord::Base
...
+		Bishop.create(color: "white", x_coord: 6, y_coord: 7)
+		Rook.create(color: "white", x_coord: 7, y_coord: 7)
+		Pawn.create(color: "white", x_coord: 0, y_coord: 6)
+		Pawn.create(color: "white", x_coord: 1, y_coord: 6)	
+		Pawn.create(color: "white", x_coord: 2, y_coord: 6)
+		Pawn.create(color: "white", x_coord: 3, y_coord: 6)
+		Pawn.create(color: "white", x_coord: 4, y_coord: 6)
+		Pawn.create(color: "white", x_coord: 5, y_coord: 6)
+		Pawn.create(color: "white", x_coord: 6, y_coord: 6)
+		Pawn.create(color: "white", x_coord: 7, y_coord: 6)
```
Comment (kenmazaika) [inline app/models/game.rb]: Cool, cool.  This piece creation logic looks exactly like I was expecting.

File: app/models/game.rb [hunk 4]
```diff
@@ -1,7 +1,42 @@
 class Game < ActiveRecord::Base
   has_many :users
   has_many :pieces
-
   delegate :kings, :queens, :bishops, :knights, :rooks, :pawns, to: :pieces
-
+	
+	def initialize
```
Comment (jcrides) [inline app/models/game.rb]: I'm pretty sure overriding initialize here is not a good idea. We want to inherit initialize from ActiveRecord::Base which does a lot of stuff for us. So maybe call this something else like populate or populate_board.

**Comment by helloramon** [comment]
Changed, thanks for the heads up but @jcrides can ya give me some more insight why that was a bad idea? I figured conceptually, Game.new would return us a new game with our starting pieces intact. I must not be looking at the bigger picture.

**Comment by jcrides** [comment]
After rename looks good to me, merging.

**Pull request merged**